TrackletAlgorithm/TrackletEngine.h

```diff
@@ -207,7 +206,6 @@ void TrackletEngine(
           	  // pT cut
           	  ap_uint<5> ptindex1=innerstubfinephi.concat(outerstubfinephi);
           	  ap_uint<5> ptindex2=ptindex1;
-          	  ap_uint<1> pt_tmp = pttable[ptindex1];
```

Comment:
more magic numbers, can these be turned into `auto`?

Reply:
I added `auto` in a bunch of places to get rid of magic numbers, and also took the opportunity to add `const` wherever it made sense.